js/exporter/jspdf/export.js

```diff
@@ -125,24 +123,23 @@ export const Export = {
         }
     },
 
-    setColumnWidths: function(autoTableOptions, columns, keepColumnWidths) {
-        const columnStyles = autoTableOptions.columnStyles;
-
-        for(let i = 0; i < columns.length; i++) {
-            columnStyles[i] = columnStyles[i] || {};
-
-            const columnWidth = columns[i].gridColumn.width;
-            if(keepColumnWidths && (typeof columnWidth === 'number') && isFinite(columnWidth)) {
-                columnStyles[i].cellWidth = this.convertPixelsToPoints(columnWidth);
-            } else {
-                columnStyles[i].cellWidth = 'auto';
-            }
+    getDefaultPdfColumnWidths(autoTableWidth, columnWidths) {
+        if(isNumeric(autoTableWidth)) {
+            const tableWidth = columnWidths.reduce((a, b) => { return a + b; });
+            return columnWidths.map((columnWidth) => {
+                return autoTableWidth * columnWidth / tableWidth;
```

Comment:
In this place and other we use the follows arrow function consruction: 
(params) => { return value}. Did you already think about deleting the redundant { return ... }? So, you code may be as (params) => value. This notation is more short.

Reply:
done